CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # CHANGELOG
 
 ## Unreleased
+- Set the `sentry-trace` header when using the tracing middleware (#1331)
```

Comment:
Nitpick: please add a blank line

```suggestion

- Set the `sentry-trace` header when using the tracing middleware (#1331)
```

Reply:
I squashed my commits and addressed this in 3e61f2c